SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorToolbar.cpp

```diff
@@ -833,6 +1105,18 @@ void FSpatialGDKEditorToolbarModule::OpenLaunchConfigurationEditor()
 	ULaunchConfigurationEditor::LaunchTransientUObjectEditor<ULaunchConfigurationEditor>(TEXT("Launch Configuration Editor"), nullptr);
 }
 
+void FSpatialGDKEditorToolbarModule::LaunchOrShowDeployment()
+{
+	if (CanLaunchDeployment())
+	{
+		OnLaunchDeployment();
```

Comment:
Think we should rename this to `OnLaunchCloudDeployment` and `CanLaunchCloudDeployment` for explicitness